samples/web-workers/ts/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "samples-web-workers-ts",
+  "private": true,
+  "version": "1.0.0",
+  "description": "A simple example of using Azure SDK in Web Workers",
+  "main": "index.js",
+  "scripts": {
+    "prestart": "npm install",
+    "start": "parcel index.html"
+  },
+  "author": "",
+  "license": "MIT",
+  "devDependencies": {
+    "@types/dotenv": "^8.2.0",
```

Comment:
Don't we need `dotenv` in dependencies? Also I don't see `dotenv` being used.

Reply:
Parcel will include `dotenv` transitively and handles calling it at the right time, so I think we're good. I should remove the types definition though since it's not needed. 

```bash
ts 11067-web-worker-storage % npm ls dotenv
samples-web-workers-ts@1.0.0 /home/mleger/workspace/azure-sdk-for-js/master/samples/web-workers/ts
└─┬ parcel-bundler@1.12.5
  └── dotenv@5.0.1 
```